src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/PropagateCancellationTokensWhenPossible.cs

```diff
@@ -32,7 +32,7 @@ public abstract class PropagateCancellationTokensWhenPossibleAnalyzer : Diagnost
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray<DiagnosticDescriptor>.Empty;
-            //ImmutableArray.Create(Rule);
+        //ImmutableArray.Create(Rule);
```

Comment:
nit: remove comment

Reply:
In the past we have had instances where developers have forgotten to update `SupportedDiagnostics` when implementing a new analyzer, and unnecessarily spent multiple investigation cycles. I'll retain the comments as it doesn't do much harm.